src/components/Categories/Categories.js

```diff
@@ -1,10 +1,21 @@
-import React from 'react';
-import styles from './Categories.module.css';
+/* eslint-disable */
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import styles from "./Categories.module.css";
+import { checkStatus } from "../../redux/categories/Categories";
 
```

Comment:
- [ ] An error occurs when the page loads because you have mixed up the naming of the module. You have `Categories` instead of `/categories`. Please fix it 👍 
![image](https://user-images.githubusercontent.com/46839548/194675718-88888ef8-40b7-438b-be7a-1e98d820de31.png)